### DISCUSSION TITLE
Thehive elast alert

### DISCUSSION THREAD

**Discussion by adilraad2001** [jertel/elastalert2#1448] [Q&A] [answered]
Hello Guys i have Question
How i can send in the alert to the hive the url of  of the event in kibana in the description

**Comment by jertel** [comment]
This is already covered in the ElastAlert 2 documentation. [Search for kibana](https://elastalert2.readthedocs.io/en/latest/search.html?q=kibana&check_keywords=yes&area=default).

**Reply by adilraad2001** [reply]
i don't find a way to send the kibana url of the event to the hive

is there any way to make the priority value of the rules to a severity of alerts

**Reply by jertel** [reply, marked as answer]
I don't think the _TheHive_ alerter supports a dynamic severity value, if that's what you're asking.
